Zend/zend_ini_scanner.l

```diff
@@ -359,13 +359,15 @@ NEWLINE	("\r"|"\n"|"\r\n")
 TABS_AND_SPACES [ \t]
 WHITESPACE [ \t]+
 CONSTANT [a-zA-Z_][a-zA-Z0-9_]*
-LABEL [^=\n\r\t;&|^$~(){}!"\[]+
+LABEL [^=\n\r\t:;&|^$~(){}!"\[]+
```

Comment:
Could this somehow be a breaking change? I wonder if people use `:` as key?

I can make a `LABEL` variant specific to variable names, if it would be better.

Reply:
Yes, this is a BC break. You might not need this if you move `<ST_VARNAME>{COLON_DASH}` before `<ST_VARNAME>{LABEL}`.

Reply:
I couldn't get lex to prefer `<ST_VARNAME>{COLON_DASH}`, it seems it will choose the longest match, if even it's not the first to appear.

To simplify this while avoiding the breakage, I simply changed the fallback separator from `:-` to `!`, I wonder if it would be a bad idea?